src/Pos/Client/CLI/Util.hs

```diff
@@ -0,0 +1,14 @@
+module Pos.Client.CLI.Util
+       ( printFlags
+       ) where
+
+import           Universum
+
+import           Pos.Constants       (isDevelopment)
+import           Pos.Util            (inAssertMode)
+printFlags :: IO ()
```

Comment:
@gromakovsky Agree! And thanks for fixing typo with https://github.com/input-output-hk/cardano-sl/commit/9449b72caab8d2006baa62c4edb0cd9a381bf4a4